ui_test/Cargo.toml

```diff
@@ -14,3 +14,7 @@ crossbeam = "0.8.1"
 lazy_static = "1.4.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+
+[features]
+# Doesn't do anything, but the miri script wants to pass the same flags to ui_test and miri itself
```

Comment:
The flags it passes are `--all-features` though so why is this needed?

Reply:
You are correct. I was trying to pass a specific feature before.